pkg/api/validation/validation.go

```diff
@@ -3405,11 +3405,11 @@ func validateResourceName(value string, fldPath *field.Path) field.ErrorList {
 
 	if len(strings.Split(value, "/")) == 1 {
 		if !helper.IsStandardResourceName(value) {
-			return append(allErrs, field.Invalid(fldPath, value, "must be a standard resource type or fully qualified"))
+			allErrs = append(allErrs, field.Invalid(fldPath, value, "must be a standard resource type or fully qualified"))
```

Comment:
The point of returning here was that no further validation is appropriate because the string is fundamentally invalid.  The fact that we don't have any other checks makes this less obvious, but assume we tested that it was all lower-case or something... I think the right error is to say "this is not a format we can handle" rather than "this is not a format we can handle, and by the way it has to be lower-case".